src/condor_utils/submit_utils.cpp

```diff
@@ -1622,89 +1622,26 @@ int SubmitHash::SetStderr()
 	return 0;
 }
 
-
-class SubmitHashEnvFilter 
+// filter for use with Env::Import that can filter based on StringList patterns
+// and on values that cannot be expressed in the given env format
+class SubmitHashEnvFilter : public WhiteBlackEnvFilter
 {
 public:
-	SubmitHashEnvFilter( Env &env, bool env1, bool env2 )
-			: m_env(env), m_env1( env1 ), m_env2( env2 ) {
-		return;
-	};
+	bool m_env1=false;
+	SubmitHashEnvFilter(bool env1) : m_env1(env1) { };
```

Comment:
Do we want an argument to this ctor that is the default white/black list that we can pass up to the parent ctor?

Reply:
seems reasonable